src/main/java/core/basesyntax/processing/FruitReport.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.processing;
```

Comment:
[All services should be in the service package, also like all his implementations should be in impl package ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop#dont-keep-all-logic-in-a-single-package)